src/security_utilities_rust/src/microsoft_security_utilities_core/identifiable_secrets.rs

```diff
@@ -177,61 +254,66 @@ pub fn generate_common_annotated_test_key(
 
         let key_bytes_length = key_bytes.len();
 
-        key_bytes[key_bytes_length - 27] = reserved_bytes[2];
-        key_bytes[key_bytes_length - 26] = reserved_bytes[1];
-        key_bytes[key_bytes_length - 25] = reserved_bytes[0];
+        key_bytes[key_bytes_length - 25] = reserved_bytes[2];
```

Comment:
>25 [](http://example.com/codeflow?start=37&length=2)

wow! we were quite out of date I see. :)

Reply:
@michaelcfanning Indeed. This also needs a plan of action: whenever the C# code is updated, the Rust code must reflect a semantically equivalent update.